modules/core/02-client/client/cli/query.go

```diff
@@ -48,6 +49,10 @@ func GetCmdQueryClientStates() *cobra.Command {
 				return err
 			}
 
+			for _, cs := range res.ClientStates {
+				maybeDecodeWasmData(clientCtx, cs.ClientState)
```

Comment:
Wouldn't it be overriding and only the last one would be valid?

Reply:
No, why?

Reply:
oh, I see the cs is new. Sorry, my bad